AppCenterCrashes/AppCenterCrashesTests/MSErrorLogFormatterTests.mm

```diff
@@ -501,8 +501,12 @@ - (void)assertIsCrashProbeReportValidConverted:(NSString *)filename {
   NSArray *images = crashReport.images;
   for (PLCrashReportBinaryImageInfo *image in images) {
     if (image.codeType != nil && image.codeType.typeEncoding == PLCrashReportProcessorTypeEncodingMach) {
-      assertThat(errorLog.primaryArchitectureId, equalTo(@(image.codeType.type)));
-      assertThat(errorLog.architectureVariantId, equalTo(@(image.codeType.subtype)));
+      XCTAssertEqual(errorLog.primaryArchitectureId.unsignedLongLongValue, image.codeType.type,
+                     @"Report: %@, Image: %@", filename, [image.imageName lastPathComponent]);
+      // It might be not equal valid architectures is wider than actual compile targets.
```

Comment:
Uncomment?

Reply:
No. This is a comment WHY this check is commented/disabled.